ballerina/Package.md

```diff
@@ -1,22 +1,117 @@
-Connects to MongoDB from ballerina 
+## Overview
 
-## Package overview
+[MongoDB](https://docs.mongodb.com/v4.2/) is a general purpose, document-based, distributed database built for modern application developers and for the cloud era. MongoDB offers both a Community and an Enterprise version of the database.
 
-The `ballerinax/mongodb` is a [Ballerina](https://ballerina.io/) connector for MongoDB.
+The `ballerinax/mongodb` package offers APIs to connect to MongoDB servers and to perform various operations including CRUD operations, indexing, and aggregation. The Ballerina MongoDB connector uses the MongoDB Java Sync library underneath. It is compatible with MongoDB 3.6 and later versions.
 
-The MongoDB connector allows you to connect to a MongoDB database from Ballerina and perform the MongoDB CRUD operations.
+## Setup guide
 
-## Compatibility
+To use the MongoDB connector, you need to have a MongoDB server running and accessible. For that, you can either install MongoDB locally or use the [MongoDB Atlas](https://www.mongodb.com/cloud/atlas/register), the cloud offering of the MongoDB.
 
-|                           | Version                     |
-|---------------------------|-----------------------------|
-| Ballerina Language        | Swan Lake 2201.7.0          |
-| Mongo DB                  | 4.2.0                       |
+### Setup a MongoDB server locally
```

Comment:
Is our connector is restricted to be used with any specific mongo-db version ? If so, shall we add a compatibility matrix to this section ?

Reply:
Compatibility is mentioned in the spec. I'll add it here.